webhooks-extension/test/util.sh

```diff
@@ -126,69 +126,105 @@ function install_tekton() {
-# If it doesn't : check that the user is ok for us to clone and ko apply it
-# If it does: check that the user is ok for us to cd into and ko apply it
+# An option to clone dashboard is provided.
+# Installation process is either:
+# kubectl applying current release via github uri
+# kubectl applying gcr nightly build yaml
+# kubectl applying curled nightly build
+# ko apply development version
 
-function install_dashboard() { 
-  if [ -z "$1" ]; then
-        echo "Usage ERROR for function: install_dashboard [namespace]"
-        echo "Missing [namespace]"
-        exit 1
+function install_dashboard() {
+  if [ -z "$1" ] || [ -z "$2" ]; then
+      echo "Usage ERROR for function: install_webhooks_extension [namespace] [version] <sleepTime>"
+      [ -z "$1" ] && echo "Missing [namespace]"
+      [ -z "$2" ] && echo "Missing [version]"
+      exit 1
   fi
-  namespace="$1"
-  check GOPATH
-  check KO_DOCKER_REPO docker.io/your_docker_id
-
-  dashboard_dir=$GOPATH/src/github.com/tektoncd/dashboard 
-  if [ -d $dashboard_dir ]; then 
-    echo "Dashboard source detected at $dashboard_dir"
-    read -p "Do you wish to ko apply the current source? " -n 1 -r apply
+
+  version=$2
+  # Current Release
+  if [[ $version == "1" ]]; then
+    kubectl apply --filename https://github.com/tektoncd/dashboard/releases/download/v0.2.0/release.yaml
   else
-    echo "Dashboard source not detected in $dashboard_dir"
-    read -p "Do you wish to git clone and ko apply the master version? " -n 1 -r cloneAndApply
+    # Check if user has dashboard source cloned
+    dashboard_dir=$GOPATH/src/github.com/tektoncd/dashboard
+    if [ -d $dashboard_dir ]; then
+      # Notify and depending on [version] we will ko apply or kubectl apply gcr yaml
+      echo -e "\n\nDashboard source detected at $dashboard_dir"
+    else
+      # If not, notify and ask if he wants to clone it
+      echo -e "\n\nDashboard source not detected in $dashboard_dir"
+      if [[ $version != "3" ]]; then
+        read -p "Do you wish to git clone and ko apply the master version? " -n 1 -r cloneAndApply
+      fi
+      if [[ $cloneAndApply =~ ^[Yy]$ ]] || [[ $version == "3" ]]; then
+        # Dashboard source cloned
+        pushd $GOPATH/src/github.com/tektoncd/
+        git clone https://github.com/tektoncd/dashboard.git
+        popd
+      else
+        # User does not want to clone repo therefore if user chooses to use the
+        # nightly build then the only way to get it is by curling it
+        echo -e "\nCurling and applying nightly build...."
+        curl -L https://github.com/tektoncd/dashboard/releases/download/v0/gcr-tekton-dashboard.yaml | kubectl apply -f -
+      fi
+    fi
+
+    # At this point has the dashboard cloned and the only thing left to do is
+    # ko apply or kubectl apply gcr yaml
+    if [[ $version == "3" ]]; then
+      pushd $GOPATH/src/github.com/tektoncd/dashboard
+      docker login
+      npm install
```

Comment:
Should be `npm ci`? You use it later so wondering if npm install is intentional here

Reply:
definitely, thanks Adam!